v2.1/import-data.md

```diff
@@ -77,10 +77,89 @@ For reference, CockroachDB uses these defaults:
 - `[port]`: **26257**
 - `[user]`: **root**
 
+
+## Import from MySQL Dump
+
+{% include experimental-warning.md %}
+
+<span class="version-tag">New in v2.1:</span> This section has instructions for getting data from MySQL dump files into CockroachDB using [`IMPORT`](import.html). It uses the [employees data set](https://github.com/datacharmer/test_db) that is also used in the [MySQL docs](https://dev.mysql.com/doc/employee/en/).
+
+- [Step 1. Dump the MySQL database](#step-1-dump-the-mysql-database)
+- [Step 2. `IMPORT` the dump files](#step-2-import-the-dump-files)
+
+### Step 1. Dump the MySQL database
+
+Make sure your MySQL user has permission to execute [`SELECT INTO OUTFILE`](https://dev.mysql.com/doc/refman/8.0/en/select.html), as required by [`mysqldump`](https://dev.mysql.com/doc/refman/8.0/en/mysqldump-sql-format.html). Then issue a command like the following (modified as needed for your environment):
```

Comment:
I believe this only applies to the `--tab` output mode of `mysqldump`, so I think since we're only showing the SQL dump, this note and the notes below about error 1290, don't apply any more.

I think we'll want a separate section for outfile, where we can put these warnings, along with recommendation for escape chars, etc but I think that is pretty low-priority: the SQL format is the default dump format and is better supported by cockroach, so I think it makes sense to focus on it initially.

Reply:
> I believe this only applies to the --tab output mode of mysqldump, so I think since we're only showing the SQL dump, this note and the notes below about error 1290, don't apply any more.

Fixed, thanks

> I think we'll want a separate section for outfile, where we can put these warnings, along with recommendation for escape chars, etc but I think that is pretty low-priority: the SQL format is the default dump format and is better supported by cockroach, so I think it makes sense to focus on it initially.

Great!